### PR TITLE
refactor(architecture): land 10of10 remediation surfaces

### PR DESCRIPTION
**Pull request by jamesryancooper**
## What
Promotes the architecture 10/10 remediation packet into durable repo surfaces: registry-backed topology docs, explicit evidence/promotion/support-proof contracts, authority-engine modularization, retained closure evidence, and archived packet lineage.

## Why
The repo had the target-state architecture directionally present but still lacked a single machine-readable topology registry, generalized proof/receipt contracts, boundary-coverage validation, and closure-grade evidence outside the proposal workspace.

No-Issue: user-directed architecture remediation and publish flow.

## How
Expanded the architecture contract registry into the machine-readable SSOT, reduced active docs to registry-backed steady-state roles, added the missing runtime/constitutional/governance contracts and validators, split `authority_engine` into concept-aligned modules behind the existing public facade, retained 10of10 validation evidence, archived the proposal packet, and refreshed extension publication so inherited validators stayed green after the doc/obligation changes.

## Profile Selection Receipt
- Semantic version source(s): `/.octon/framework/constitution/charter.yml`, `/.octon/instance/charter/workspace.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `atomic`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): broad architecture/control-plane change; no external downtime; rollback is commit/PR revert; closure evidence retained under `state/evidence/validation/architecture/10of10-remediation/**`
- Tie-break status: none
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): n/a

## Implementation Plan
- Workstreams and scope: registry/docs consolidation; new contracts/specs/policies; authority-engine modularization; validator and workflow wiring; retained evidence/read-model publication; packet archival
- Public interfaces/types/contracts affected: architecture contract registry, evidence/fail-closed obligations, runtime spec contracts, governance receipt/proofing policies, authority-engine module layout
- Test scenarios: registry/doc consistency, authorization-boundary coverage, evidence completeness, input/generated non-authority, promotion receipt posture, support proofing, operator read-model traceability, inherited architecture/runtime checks, authority-engine crate tests
- Assumptions/defaults: existing live bounded-admitted support model remains authoritative; disclosure v2 remains canonical; extension publication refresh is required to keep inherited validation current after doc/obligation changes

## Impact Map (code, tests, docs, contracts)
- Code: authority engine modularization, kernel request flag fixes, workflow/CI wiring
- Tests: `cargo test -p octon_authority_engine`
- Docs: `/.octon/README.md`, `specification.md`, `START.md`, `instance/ingress/AGENTS.md`, ADRs 094-100
- Contracts: architecture registry, new retention/authority/assurance/runtime contracts, governance receipt/proofing policies, evidence/fail-closed obligations

## Compliance Receipt
- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations
none

## Tradeoffs
Large diff because the work promotes the full remediation packet, archives the lineage packet, and carries the evidence/read-model surfaces needed for closure. The authority-engine extraction still emits unused-import warnings, but behavior and tests are intact.

## Testing
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-architecture-contract-registry.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-authorization-boundary-coverage.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-evidence-completeness.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-generated-non-authority.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-input-non-authority.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-promotion-receipts.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-support-target-proofing.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-operator-read-models.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-runtime-docs-consistency.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-architecture-conformance.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-mission-runtime-contracts.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-phase5-adapter-support-target-hardening.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-runtime-target-parity.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-extension-publication-state.sh`
- `cargo test -p octon_authority_engine`

## Rollout
n/a

## Convivial Purpose Check
- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric
- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan: revert the PR to restore the prior architecture/doc/runtime surfaces
- Rollback handle: commit `0136d3a815cd1ffceb8dc5a40560fd7497fdb1ec`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model
- OpenAPI/JSON-Schema changes: new `evidence-store-v1`, `promotion-receipt-v1`, `support-target-proof-bundle-v1`
- Threat model update/link: fail-closed and promotion/support-proof posture tightened in constitutional obligations and validators

## Observability and Performance
- Traces/logs/metrics for changed flows: retained 10of10 evidence root plus existing run/control/disclosure receipts
- Representative traces for high risk changes: representative run evidence in `evidence-store/completeness.yml`
- Performance or bundle impact: no material runtime performance target claimed; larger authored/evidence surface only

## License and Provenance
- New dependencies and licenses: none
- Generated code/templates provenance notes: retained operator/read-model projections and archived proposal packet lineage are generated or archived intentionally; proposal remains non-authoritative lineage

## Checklist
- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [ ] Reviewer feedback addressed with fix, commit, push, reply; reviewer-owned threads left for reviewer or maintainer confirmation

## Screenshots / Notes
No visual changes. The branch also refreshes extension publication receipts/state because the inherited runtime-effective validator depends on those digests staying current after architecture doc and obligation changes.

### CODE REVIEW

File: .github/workflows/runtime-binaries.yml [hunk 1]
```diff
@@ -73,6 +73,13 @@ jobs:
       - name: Install Rust toolchain
         uses: dtolnay/rust-toolchain@631a55b12751854ce901bb631d5902ceb48146f7 # stable
 
+      - name: Install yq
+        shell: bash
+        run: |
+          set -euo pipefail
+          sudo wget -qO /usr/local/bin/yq https://github.com/mikefarah/yq/releases/download/v4.50.1/yq_linux_amd64
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .github/workflows/runtime-binaries.yml]: Addressed in `bac934f64`. The `runtime-binaries` matrix now installs `yq` per runner OS instead of assuming Linux + `sudo`, so non-Linux release targets no longer fail in that setup step.

File: .octon/framework/assurance/runtime/_ops/scripts/validate-operator-read-models.sh [hunk 2]
```diff
@@ -0,0 +1,460 @@
...
+}
+
+materialize_projection_if_missing() {
+  local target="$1"
+  local body="$2"
+  if [[ -f "$target" ]]; then
+    return 0
+  fi
+  mkdir -p "$(dirname "$target")"
+  printf '%s\n' "$body" > "$target"
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/assurance/runtime/_ops/scripts/validate-operator-read-models.sh]: Addressed in `bac934f64`. The validator no longer materializes missing projections; it now only verifies the committed/generated artifacts and fails if required projections are absent.
